pom.xml

```diff
@@ -1848,5 +1848,142 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>quick</id>
+            <activation>
+                <property>
+                    <name>quick</name>
```

Comment:
CI builds could use this for their MAVEN_FAST_INSTALL opts.

Reply:
Done!

Reply:
taken back per https://github.com/trinodb/trino/pull/10519#discussion_r781285821